app/views/servers/_form.html.erb

```diff
@@ -19,6 +19,11 @@
     <%= f.label :url %><br>
     <%= f.text_field :url, class: "form-control" %>
   </div>
+  <div class="field form-group">
+    <label>
+    <%= f.check_box :kind, {}, "graphite", "prometheus" %>Graphite server
```

Comment:
I think it would be prettier if this were a dropdown ("Prometheus", "Graphite", with the former being the default). But check box works for now too